.idea/modules.xml

```diff
@@ -4,6 +4,7 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/PixelPerfect.iml" filepath="$PROJECT_DIR$/PixelPerfect.iml" />
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/pixelperfect-client.iml" filepath="$PROJECT_DIR$/pixelperfect-client.iml" />
```

Comment:
My IDE cannot locate this module since it has not created it, leaving the project "out of sync".